experimental/packages/sdk-logs/src/Logger.ts

```diff
@@ -53,4 +54,22 @@ export class Logger implements logsAPI.Logger {
      */
     logRecordInstance._makeReadonly();
   }
+
+  public emitEvent(eventRecord: logsAPI.EventRecord): void {
+    const attributes = eventRecord.attributes || {};
+    attributes['event.name'] = eventRecord.name;
```

Comment:
Nit: (use semantic conventions)

```suggestion
    attributes[ATTR_EVENT_NAME] = eventRecord.name;
```

https://github.com/open-telemetry/opentelemetry-js/blob/main/semantic-conventions/src/experimental_attributes.ts#L2431